opencti-platform/opencti-graphql/src/domain/subType.ts

```diff
@@ -19,11 +19,11 @@ export const queryDefaultSubTypes = async (context: AuthContext, user: AuthUser,
     const types = schemaTypesDefinition.get(ABSTRACT_STIX_DOMAIN_OBJECT).filter((n) => n.includes(search ?? ''));
     const finalResult = R.pipe(
       sortByLabel,
-      R.map((n) => ({ node: { id: n, label: n } })),
-      R.append({ node: { id: ABSTRACT_STIX_CORE_RELATIONSHIP, label: ABSTRACT_STIX_CORE_RELATIONSHIP } }),
-      R.append({ node: { id: STIX_SIGHTING_RELATIONSHIP, label: STIX_SIGHTING_RELATIONSHIP } }),
-      R.append({ node: { id: ABSTRACT_STIX_CYBER_OBSERVABLE, label: ABSTRACT_STIX_CYBER_OBSERVABLE } }),
-      R.append({ node: { id: ENTITY_HASHED_OBSERVABLE_ARTIFACT, label: ENTITY_HASHED_OBSERVABLE_ARTIFACT } }),
+      R.map((n) => ({ node: { id: n, label: n, overridable: true } })),
```

Comment:
Can you explain the new overridable value?

Reply:
Overridable determines if the entity type appears on the 'Entities Override' tab on the role edit drawer. This line ensures the entity types can be selected for overriding capabilities.